app/src/ui/branches/pull-request-list.tsx

```diff
@@ -206,8 +207,12 @@ export class PullRequestList extends React.Component<
 function createListItems(
   pullRequests: ReadonlyArray<PullRequest>
 ): IFilterListGroup<IPullRequestListItem> {
+  function getSubtitle(pr: PullRequest) {
```

Comment:
:art: Does this need to be defined inline? I think it can live outside `createListItems`, not sure if I'm missing some context...

Reply:
No we don't need to inline this. I will move it out.